### PR TITLE
feature/contactform

### PR DESCRIPTION
**Pull request by Gijs123**
contact form

### CODE REVIEW

File: resources/views/contact/sentSuccessfully.blade.php [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@extends('layout.app')
+
+
+@section('title', 'contact form send successfully')
+
+@section('content')
+<h1> Your form has been sent successfully <h1>
```
Comment (HonkingGoose) [inline resources/views/contact/sentSuccessfully.blade.php]: I think a `<h2>` would be better here.

File: resources/views/contact/form.blade.php [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        <div class="form-group">
+            <label class="control-label col-md-4" for="message">Message:</label>
+                <div class="col-md-8">
+                    <input class="form-control" id="Message_id" type="text" name="message" class="form-control @error('message') is-invalid @enderror" value="{{ old('message') }}" required autocomplete="message" autofocus>
+                </div>
+        </div>
+  
+        <br />
+        <div class="form-group" align="center">
+            <!-- <input type="hidden" name="action" id="action" value="Add" />
```
Comment (HonkingGoose) [inline resources/views/contact/form.blade.php:51]: Is this commented out code needed later or not?

File: app/Http/Controllers/ContactFormController.php [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        return view('contact.form');
+    }
+
+    public function send(Request $request)
+    {
+
```
Comment (HonkingGoose) [inline app/Http/Controllers/ContactFormController.php]: Remove unnecessary white-space.

**Review by HonkingGoose — CHANGES_REQUESTED** [review]

**Review by mnk87 — DISMISSED** [review]

File: resources/views/contact/sentSuccessfully.blade.php (hunk 1 above)
Comment (HonkingGoose) [reply resources/views/contact/sentSuccessfully.blade.php]: @mnk87 What do you think?

**Comment by HonkingGoose** [comment]
@mnk87 We need to check if this works together with the new changes on `master`. I've pushed the changes from `master` into this branch.

So both of you @mnk87 and @Gijs123 need to do `git pull` on this branch, so that the new changes can be tested on your local computer.

---

EDIT:

I made some more changes, I'm done adding commits. So you can continue working @Gijs123.

**Review by HonkingGoose — CHANGES_REQUESTED** [review]
There is no input validation on the contact form.

Example:

- "First name": 123
- "Last name": 123
- "Room number": 1
- "Choose a category": complaint.
- "Message": This is a TEST.

This input is put in the database without any complaints or errors from Laravel or the database.

---

On the bright side, the contact form does work... 😄

**Review by HonkingGoose — APPROVED** [review]

**Pull request merged**